app/src/main/java/cm/aptoide/pt/notification/CampaignNotificationWorker.kt

```diff
@@ -0,0 +1,44 @@
+package cm.aptoide.pt.notification
+
+import android.content.Context
+import android.util.Log
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import cm.aptoide.pt.crashreports.CrashReport
+import cm.aptoide.pt.sync.Sync
+import cm.aptoide.pt.sync.SyncScheduler
+import cm.aptoide.pt.sync.alarm.AlarmSyncScheduler
+import cm.aptoide.pt.sync.alarm.SyncStorage
+
+class CampaignNotificationWorker(context: Context, workerParameters: WorkerParameters,
```

Comment:
As discussed, we probably should rename this to something like "NotificationWorker"